src/block.rs

```diff
@@ -71,6 +71,13 @@ impl bindings::bech32_nprofile {
     pub fn pubkey(&self) -> &[u8; 32] {
         unsafe { &*(self.pubkey as *const [u8; 32]) }
     }
+
+    pub fn relays_to_strings(&self) -> Vec<String> {
+        self.relays.relays[0..(self.relays.num_relays as usize)]
+            .iter()
+            .map(|block| block.as_str().to_string())
+            .collect()
+    }
```

Comment:
this is not correct, no nostrdb api should require copying data. we probably want to make an iterator for this